feature-libs/product-configurator/rulebased/components/overview-notification-banner/configurator-overview-notification-banner.component.ts

```diff
@@ -40,14 +40,28 @@ export class ConfiguratorOverviewNotificationBannerComponent {
       //In case overview carries number of issues: We take it from there.
       //otherwise configuration's number will be accurate
       if (configuration.overview?.totalNumberOfIssues) {
-        return configuration.overview.totalNumberOfIssues;
+        return configuration.overview.numberOfIncompleteCharacteristics != null
+          ? configuration.overview.numberOfIncompleteCharacteristics
+          : configuration.overview.totalNumberOfIssues;
       } else
         return configuration.totalNumberOfIssues
           ? configuration.totalNumberOfIssues
           : 0;
     })
   );
 
+  numberOfConflicts$: Observable<number> = this.configuration$.pipe(
+    map((configuration) => {
+      return configuration.overview?.numberOfConflicts ?? 0;
+    })
+  );
+
+  ignoreConflicts$: Observable<boolean> = this.configuration$.pipe(
```

Comment:
This better 'skipConflictsOnIssueNavigation'. And please add JS doc, because also with the changed name it's hard to grasp why we do it like this